ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -422,6 +432,36 @@ export class OntotextYasguiWebComponent {
     this.rebuild(yasguiConfiguration);
   }
 
+  /**
+   * Handler for the event fired when the copy link dialog is closed without copying the link to the clipboard.
+   */
+  @Listen('internalResourceLinkDialogClosedEvent')
+  resourceLinkDialogClosedHandler() {
+    this.showCopyResourceLinkDialog = false;
+    this.copiedResourceLink = undefined;
+  }
+
+  /**
+   * Handler for the event fired when the copy link dialog is closed when the link is copied to the clipboard.
+   */
+  @Listen('internalResourceLinkCopiedEvent')
+  resourceLinkCopiedHandler() {
+    this.notify.emit(new NotificationMessage(NotificationMessageType.SUCCESS, this.translationService.translate('yasqe.share.copy_link.dialog.copy.message.success')));
```

Comment:
I'm a bit hesitant about this. Time will tell if this will survive in favor of the general purpose event bus. But if it does, then maybe we need some message type based on which the client can deside if he wants to show it or not.